service/src/java/org/apache/hive/service/cli/thrift/ThriftCLIService.java

```diff
@@ -334,8 +405,7 @@ public TOpenSessionResp OpenSession(TOpenSessionReq req) throws TException {
       resp.setSessionHandle(sessionHandle.toTSessionHandle());
       resp.setConfiguration(map);
       resp.setStatus(OK_STATUS);
-      ThriftCLIServerContext context =
-        (ThriftCLIServerContext)currentServerContext.get();
+      ThriftCLIServerContext context = (ThriftCLIServerContext) currentServerContext.get();
```

Comment:
Nit: Is this a formatting only change? If so, do we need it?

Reply:
Reverted.